### PR TITLE
hell no

### PR DESCRIPTION
**Pull request by ghost**
that description sucked ass and you forgot to capitalize Optifine

### CODE REVIEW

**Review by koxx12-dev — APPROVED** [review]

File: files/mods.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
       "file": "optifine_1.8.9_HD_U_M5.jar",
       "url": "https://optifine.net/download?f=optifine_1.8.9_HD_U_M5.jar",
       "display": "optifine M5",
-      "description": "optifine allows your game to be more customizable and push it to its limits while also providing an FPS boost.",
+      "description": "Optifine is an EXTREMELY lucrative modification that is required for most mods and practically all SkyBlock texture packs. It also adds an incredible amount of customizability options for Minecraft's graphics settings, from tree leaves to fog.",
```
Comment (TymanWasTaken) [inline files/mods.json:136]: What mods is optifine required for lol

Comment (MicrocontrollersDev) [reply files/mods.json:136]: neu dwarven overlay is the only one i can think of

**Comment by MicrocontrollersDev** [comment]
F in OptiFine should be cap smh

**Review by koxx12-dev — CHANGES_REQUESTED** [review]
i will marge it when u add capitalization to the line 135 , 133

File: files/mods.json (hunk 1 above)
Comment (koxx12-dev) [reply files/mods.json:136]: optifine is needed to change textures depending on the name / nbt / other stuff

**Review by nacrt — APPROVED** [review]

**Comment by koxx12-dev** [comment]
i will fix it myself

**Pull request merged**